source/backend/api/Pims.Api.csproj

```diff
@@ -2,8 +2,8 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <UserSecretsId>0ef6255f-9ea0-49ec-8c65-c172304b4926</UserSecretsId>
-    <Version>5.11.0-105.8</Version>
-    <AssemblyVersion>5.11.0.105</AssemblyVersion>
+    <Version>5.11.1-105.8</Version>
+    <AssemblyVersion>5.11.1.105</AssemblyVersion>
```

Comment:
This PR is targeting DEV branch so bumping the hotfix version is not required - please remove